src/ngx_http_lua_ssl_certby.c

```diff
@@ -880,6 +880,30 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 #endif
 }
 
+int
+ngx_http_lua_ffi_ssl_client_addr(ngx_http_request_t *r, char **addr,
```

Comment:
@smartwjw I see. Need some test cases for various different cases like unix domain socket clients.